Auflistung von DHCP-Leases und Wifi-Clients verhindern (ein bisschen mehr Privatsphaere)
--- a/luci/modules/admin-full/luasrc/view/admin_status/index.htm
+++ b/luci/modules/admin-full/luasrc/view/admin_status/index.htm
@@ -16,8 +16,10 @@ You may obtain a copy of the License at
 	require "luci.tools.status"
 
 	local has_ipv6 = luci.fs.access("/proc/net/ipv6_route")
-	local has_dhcp = luci.fs.access("/etc/config/dhcp")
-	local has_wifi = luci.fs.stat("/etc/config/wireless")
+	-- OPENNET: wir deaktivieren die Darstellung fuer dhcp und wifi
+	-- damit die Clients und assoziierten WLAN-Geraete nicht passwortfrei von aussen sichtbar sind
+	local has_dhcp = false
+	local has_wifi = false
 	      has_wifi = has_wifi and has_wifi.size > 0
 	local _, _, memtotal, memcached, membuffers, memfree, _, swaptotal, swapcached, swapfree = luci.sys.sysinfo()
 	local has_swap
